### PR TITLE
Reader: Try out excerpts with line breaks in them

### LINKED ISSUE #10067: Try allowing breaks in excerpts
We're currently removing breaks from excerpts in cards. Let's try allowing them and seeing how things look. Can we continue to disallow double breaks? IE, allow a single break, but not more than one in a row.

### PR DESCRIPTION
**Pull request by blowery**
Fixes #10067

### CODE REVIEW

**Comment by blowery** [comment]
This is a trial to see how we like how it appears in the Reader.

**Comment by fraying** [comment]
I think we should do this. Here's what convinced me:
https://wordpress.com/read/blogs/68015336
vs
https://calypso.live/read/blogs/68015336?branch=try/reader/breaks-in-excerpts

What do y'all think?

File: client/blocks/reader-post-card/index.jsx [hunk 1]
```diff
@@ -159,7 +159,13 @@ export default class RefreshPostCard extends React.Component {
 								<a className="reader-post-card__title-link" href={ post.URL }>{ title }</a>
 							</h1>
 						</AutoDirection>
-						{ showExcerpt && <AutoDirection><div className="reader-post-card__excerpt">{ post[ excerptAttribute ] }</div></AutoDirection> }
+						{ showExcerpt && (
+								<AutoDirection>
+									<div className="reader-post-card__excerpt"
+										dangerouslySetInnerHTML={ { __html: post[ excerptAttribute ] } } // eslint-disable-line react/no-danger
+									></div>
```
Comment (omarjackman) [inline client/blocks/reader-post-card/index.jsx]: Don't need the trailing `</div>` :smile:

**Comment by samouri** [comment]
I like it too

**Comment by samouri** [comment]
@bluefuton we should definitely clean that up.
I don't think we should do it yet though because its rabbit-holey and this is a tiny useful change.

I'm making an issue now to clean it up: https://github.com/Automattic/wp-calypso/issues/10109

**Pull request merged**